gcp_variant_transforms/libs/bigquery_util.py

```diff
@@ -191,3 +195,83 @@ def get_avro_type_from_bigquery_type_mode(bigquery_type, bigquery_mode):
     return [avro_type, AvroConstants.NULL]
   else:
     return avro_type
+
+def update_bigquery_schema_on_append(schema_fields, output_table):
+  # type: (bool) -> None
+  # if table does not exist, do not need to update the schema.
+  # TODO (yifangchen): Move the logic into validate().
+  output_table_re_match = re.match(
+      r'^((?P<project>.+):)(?P<dataset>\w+)\.(?P<table>[\w\$]+)$',
+      output_table)
+  credentials = GoogleCredentials.get_application_default().create_scoped(
+      ['https://www.googleapis.com/auth/bigquery'])
+  client = bigquery.BigqueryV2(credentials=credentials)
+  try:
+    project_id = output_table_re_match.group('project')
+    dataset_id = output_table_re_match.group('dataset')
+    table_id = output_table_re_match.group('table')
+    existing_table = client.tables.Get(bigquery.BigqueryTablesGetRequest(
+        projectId=project_id,
+        datasetId=dataset_id,
+        tableId=table_id))
+  except exceptions.HttpError:
+    return
+
+  new_schema = bigquery.TableSchema()
+  new_schema.fields = get_merged_field_schemas(existing_table.schema.fields,
+                                               schema_fields)
+  existing_table.schema = new_schema
+  try:
+    client.tables.Update(bigquery.BigqueryTablesUpdateRequest(
+        projectId=project_id,
+        datasetId=dataset_id,
+        table=existing_table,
+        tableId=table_id))
+  except exceptions.HttpError as e:
+    raise RuntimeError('BigQuery schema update failed: %s' % str(e))
+
+
+def get_merged_field_schemas(
+    field_schemas_1,  # type: List[bigquery.TableFieldSchema]
+    field_schemas_2  # type: List[bigquery.TableFieldSchema]
+    ):
+  # type: (...) -> List[bigquery.TableFieldSchema]
+  """Merges the `field_schemas_1` and `field_schemas_2`.
+
+  Args:
```

Comment:
nit: Leave one empty line between `Args, Returns, and Raises`. See [one example](http://go/pystyle#383-functions-and-methods). I know this is how the old code looks like, can you also help refining them? :)